app/src/main/java/com/nextcloud/talk/utils/FileUtils.kt

```diff
@@ -27,6 +27,7 @@ object FileUtils {
     private val TAG = FileUtils::class.java.simpleName
     private const val RADIX: Int = 16
     private const val MD5_LENGTH: Int = 32
+    const val FILE_MAX_LENGTH = 146
```

Comment:
the max length is set to 146 instead 150 because of the suffix ".mp3". This is not obvious in the FileUtils class and may lead to confusion. So i suggest to define the max length inside MediaRecorderManager and call it "VOICE_MESSAGE_PREFIX_MAX_LENGTH"